### PR TITLE
cuba: disable due to checksum mismatch

### PR DESCRIPTION
**Pull request by iMichka**
Upstream does not respon anymore.
As we don't konw if they have been compromised, we prefer to
disable this formula until further news, as we lost trust in upstream.

See also #88857

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Review by SMillerDev — DISMISSED** [review]

**Comment by iMichka** [comment]
@SMillerDev you reviewed this but I don't see the green check-mark. Weird.

**Comment by SMillerDev** [comment]
Because testbot dismissed it after an unsuccessful merge.

**Comment by iMichka** [comment]
Can you approve it again / merge it? I can't as long as it is not reviewed.

**Review by SMillerDev — DISMISSED** [review]

**Comment by iMichka** [comment]
No need to pull a bottle. Can we not just merge this as-is using the UI (with rebase and merge)?

**Review by SMillerDev — APPROVED** [review]

**Comment by SMillerDev** [comment]
Oh wait, no need to let testbot do this, the bottles won't be used anyway

**Pull request merged**
